internal/provider/namespace_resource.go

```diff
@@ -119,6 +124,10 @@ var (
 		"include_cross_origin_credentials": types.BoolType,
 	}
 
+	lifecycleAttrs = map[string]attr.Type{
+		"enable_delete_protection": types.BoolType,
```

Comment:
Would you mind filing an issue for adding this field to the data source as well? Just so we are keeping track of the divergence?